tests/data/packageVersion.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace ExamplePackageVersion;
+
+// TODO: phpunit/phpunit:<50 This has to be fixed when updating to phpunit 50.x
+
+// TODO: phpunit/phpunit:<5 This has to be fixed when updating to phpunit 5.x
+// TODO: phpunit/phpunit:5.3.* This has to be fixed when updating to phpunit 5.3.*
```

Comment:
actually I've done my check the other way around:
```
phpunit/phpunit:>=10
```
I think it reads better: "please phpstan, fail when phpunit meets this dependency"

About this specific todo: `phpunit/phpunit:5.3.*`, I think it would not fail when phpunit is at 5.3.*
but it will if we have phpunit>=5.4 and phpunit<5.3